map.css

```diff
@@ -0,0 +1,24 @@
+.game {
+  display: flex;
+  height: 33%;
+  margin-top:-2em;
```

Comment:
It's a measurement relative to the size of type, which is more predictable than pixels (unless you're using actual pixels, like in images).

The (non-)spaces? Just sloppiness.